### ISSUE TITLE
running rke2 binary downloaded to alpine results in error: lstat /var/lib/rancher/rke2/agent/images: no such file or directory

### ISSUE THREAD

**Issue by dweomer** [rancher/rke2#187]
I was pulling up `rke2` in a plain, vanilla Alpine container looking to see what is in the data directory and ran into this issue when running `rke2 server`: **`lstat /var/lib/rancher/rke2/agent/images: no such file or directory`**

`$ docker run --rm -it --privileged alpine`:
```
/ # cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.12.0
PRETTY_NAME="Alpine Linux v3.12"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"

/ # apk add --update curl ca-certificates
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
(1/4) Installing ca-certificates (20191127-r4)
(2/4) Installing nghttp2-libs (1.41.0-r0)
(3/4) Installing libcurl (7.69.1-r0)
(4/4) Installing curl (7.69.1-r0)
Executing busybox-1.31.1-r16.trigger
Executing ca-certificates-20191127-r4.trigger
OK: 7 MiB in 18 packages

/ # curl --output /usr/local/bin/rke2 -fsSL https://github.com/rancher/rke2/releases/download/v1.18.4-beta3%2Brke2/rke2.linux-amd64
/ # chmod +x /usr/local/bin/rke2 

/ # rke2 server --selinux=false
WARN[0000] not running in CIS 1.5 mode                  
FATA[0000] lstat /var/lib/rancher/rke2/agent/images: no such file or directory 

/ # 
```

**Comment by dweomer** [comment]
Follow-up: after making the missing directory I ran into:
```
/ # mkdir -p /var/lib/rancher/rke2/agent/images
/ # rke2 server --selinux=false
WARN[0000] not running in CIS 1.5 mode                  
FATA[0001] GET https://index.docker.io/v2/rancher/rke2-runtime/manifests/v1.18.4-beta3-rke2: MANIFEST_UNKNOWN: manifest unknown; map[Tag:v1.18.4-beta3-rke2] 
```

Probably due to https://drone-publish.rancher.io/rancher/rke2/124/1/4 failing to push the tags

**Comment by rancher-max** [comment]
Note the exact same issues also happens on Ubuntu. This is an issue with the latest binaries (beta2 and beta3).

**Comment by dweomer** [comment]
Fixed by #188
